content/en/tracing/setup/dotnet-core.md

```diff
@@ -266,7 +266,7 @@ To manually instrument your code, add the `Datadog.Trace` [NuGet package][4] to
 
 For more details on manual instrumentation and custom tagging, see [Manual instrumentation documentation][5].
 
-Manual instrumentation is supported on .NET Framework 4.5 and above on Windows and on .NET Core 2.1, 3.0, and 3.1 on Windows and Linux.
+Manual instrumentation is supported on .NET Framework 4.5 and above on Windows, on .NET Core 2.0 and above on Windows and Linux, and on .NET 5 on Windows and Linux.
```

Comment:
Same comment about adding 2.0 and 2.2 as in `content/en/tracing/custom_instrumentation/dotnet.md`.